llvm/test/Transforms/LoopVectorize/RISCV/zext-drop-nneg.ll

```diff
@@ -0,0 +1,104 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=loop-vectorize -S -mtriple=riscv64-unknown-linux-gnu -mattr=+v < %s | FileCheck %s
```

Comment:
Can this be a target independent test?

Reply:
I tried to make target independent but could not reproduce it. Forced vector length does not trigger the crash, forced scalable too.

Reply:
Ah right, this needs a target with masked gather-scatter support. The existing tests are in `llvm/test/Transforms/LoopVectorize/X86/drop-poison-generating-flags.ll` might be good to consolidate them there.